ts/features/it-wallet/screens/ItwCredentialsCatalogScreen.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { SafeAreaView, ScrollView } from "react-native";
+import { Badge, IOStyles, Icon, VSpacer } from "@pagopa/io-app-design-system";
+import BaseScreenComponent from "../../../components/screens/BaseScreenComponent";
+import { emptyContextualHelp } from "../../../utils/emptyContextualHelp";
+import { ListItemItw } from "../components/ListItemItw";
+import ScreenContent from "../../../components/screens/ScreenContent";
+import ItwSearchBar from "../components/ItwSearchBar";
+import I18n from "../../../i18n";
+
+/**
+ * Renders a preview screen which displays a visual representation and the claims contained in the PID.
+ */
+const ItwCredentialsCatalogScreen = () => {
+  /**
+   * Renders the content of the screen if the PID is decoded.
+   * @param decodedPip - the decoded PID
+   */
+  const ContentView = () => (
```

Comment:
In the next releases the catalog could be downloaded remotely. What do you think about doing a refactor using mocked data and a renderListItem method?

Reply:
Good idea, I'll refactor this in the next commit. Thanks!